src/Maker/AdminMaker.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Maker;
+
+use Sonata\AdminBundle\Command\Validators;
+use Sonata\AdminBundle\Manipulator\ServicesManipulator;
+use Sonata\AdminBundle\Model\ModelManagerInterface;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Maker\AbstractMaker;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\DependencyInjection\Container;
+
+/**
+ * @author Gaurav Singh Faudjdar <faujdar@gmail.com>
+ */
+class AdminMaker extends AbstractMaker
+{
+    /**
+     * @var string
+     */
+    private $projectDirectory;
+
+    /**
+     * @var ModelManagerInterface[]
+     */
+    private $availableModelManagers;
+
+    public function __construct($projectDirectory, array $modelManagers = [])
+    {
+        $this->projectDirectory = $projectDirectory;
+        $this->availableModelManagers = $modelManagers;
+    }
+
+    /**
+     * Return the command name for your maker (e.g. make:report).
+     *
+     * @return string
+     */
+    public static function getCommandName(): string
+    {
+        return 'make:sonata:admin';
+    }
+
+    /**
+     * Configure the command: set description, input arguments, options, etc.
+     *
+     * By default, all arguments will be asked interactively. If you want
+     * to avoid that, use the $inputConfig->setArgumentAsNonInteractive() method.
+     *
+     * @param Command            $command
+     * @param InputConfiguration $inputConfig
```

Comment:
Please remove phpdoc when it doesn't add more information than the type hints already do